README.md

```diff
@@ -29,6 +29,7 @@ Maven and Gradle
     - [Getting Started](#getting-started)
     - [Initial Considerations](#initial-considerations)
     - [Usage](#usage)
+- [Properties Validation](#properties-validation)
```

Comment:
I would rename the section to `Property Validation` to make it sound more natural. Just a nitpick